src/Emails/AccountOnboardingInvitationEmail.php

```diff
@@ -3,8 +3,9 @@
 namespace Rhubarb\Scaffolds\Authentication\Emails;
 
 use Rhubarb\Crown\Settings\WebsiteSettings;
+use Rhubarb\Scaffolds\Authentication\Settings\ProtectedUrl;
 
-class ActivateAccountInvitationEmail extends ResetPasswordInvitationEmail
```

Comment:
I'd leave a deprecated version of all of these classes around in case someone was using them